docs/general/serializers.md

```diff
@@ -382,11 +382,26 @@ The serialized value for a given key. e.g. `read_attribute_for_serialization(:ti
 
 #### #links
```

Comment:
Do you man that `#### #links` should be `#### links`? Many `h4` headers in this document have a `#` as part of the text. I believe this is to indicate that they are instance methods. Would you like me to remove all of them? e.g. ln 238, ln 252